decko/spec/cypress/integration/edit/pointer.spec.coffee

```diff
@@ -47,7 +47,7 @@ describe 'editing pointers', () ->
     cy.visit("/Joe User+friends")
     cy.get("._add-item-link").click()
     cy.contains("Select Item")
-    cy.contains("button", "Add filter").click()
+    cy.contains("button", "More Filters").click()
```

Comment:
debatable. I think "more filters" is less scary, because users don't think they're being asked to add anything.  But "add filter" makes more sense if you have an initial setup with no filters on.